packages/framework/esm-styleguide/src/datepicker/openmrs/openmrs-date-picker.component.tsx

```diff
@@ -11,14 +12,14 @@ const DEFAULT_PLACEHOLDER = 'dd/mm/yyyy';
 
 export interface OpenmrsDatePickerProps {
   id: string;
-  labelText: string;
+  labelText: string | any;
```

Comment:
Technically, what's supported here is [this](https://github.com/carbon-design-system/carbon/blob/5078eb5204b21763244768af3b0b283551561d47/packages/react/src/components/DatePickerInput/DatePickerInput.tsx#L19). No explicit `any`s should be added to the framework unless its unavoidable.

```suggestion
  labelText: string | ReactElement;
```